types/electron-fetch/index.d.ts

```diff
@@ -0,0 +1,117 @@
+import { Readable as NodeReadableStream } from 'stream'
+
+export default function fetch (input: RequestInfo, init?: RequestInit): Promise<Response>
+
+export interface Body {
+  readonly body: NodeReadableStream | ReadableStream<Uint8Array> | null
+  readonly bodyUsed: boolean
+  arrayBuffer: () => Promise<ArrayBuffer>
+  blob: () => Promise<Blob>
+  formData: () => Promise<FormData>
+  json: () => Promise<any>
+  text: () => Promise<string>
+  buffer: () => Promise<Buffer>
+}
+export class Headers extends globalThis.Headers {}
```

Comment:
Wait where's this `globalThis` coming from ?

Reply:
Wow when does this happen https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis 🤯